frontend/lib/api/form/useErrorsAndWarnings.ts

```diff
@@ -48,3 +53,49 @@ export function useErrorsAndWarnings(
     return result;
   }, [errors, warnings, clientWarnings]);
 }
+
+//TODO: intermediate step for testing, might scrap code above.
```

Comment:
What needs to happen to resolve this TODO?

Reply:
the useErrosandwarnings hook can be removed. then move the getErrorsandwarnings to a better place.

Reply:
refactored and added tests